src/main/java/core/selenium/webdrivers/Firefox.java

```diff
@@ -0,0 +1,41 @@
+package core.selenium.webdrivers;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+
+/**
+ * Firefox class.
+ *
+ * @author Raul Choque
+ * @version 0.0.1
+ */
+public final class Firefox implements Browser {
+
+    private static Firefox firefox;
+    private WebDriver driver;
+
+    /**
+     * Constructor of Firefox class.
+     */
+    private Firefox() {
+        System.setProperty("webdriver.gecko.driver", "src/main/java/core/selenium/webdrivers/driver/geckodriver");
```

Comment:
I think It look better if you don't all the path, for that i recomment you use variable or enum.

Reply:
Thank you, this wrong was fixed.